widget/ui/src/components/Popover/Popover.tsx

```diff
@@ -16,7 +16,10 @@ export const PopoverContentComponent = (
   const { container, children, ...rest } = props;
   return (
     <RadixPopover.Portal container={container}>
-      <PopoverContainer {...rest} ref={forwardedRef}>
+      <PopoverContainer
+        onOpenAutoFocus={(e) => e.preventDefault()}
+        {...rest}
```

Comment:
This solution doesn't completely address the issue.

Reply:
It was fixed in another PR by @Ikari-Shinji-re .